.vortex/docs/.utils/VideoRecorder.php

```
@@ -302,13 +302,56 @@ public function postprocessCast(string $cast_path, ?string $workspace = NULL, bo
+      // GitHub personal access, OAuth, user-to-server, server-to-server and
+      // refresh tokens (for example 'ghp_...', 'gho_...').
+      '#\bgh[oprsu]_[A-Za-z0-9]{36,255}#',
+      // GitHub fine-grained personal access tokens ('github_pat_...').
+      '#\bgithub_pat_[A-Za-z0-9_]{22,255}#',
+      // AWS access key identifiers.
+      '#\b(?:AKIA|ASIA)[A-Z0-9]{16}#',
+    ];
+
+    $masked = preg_replace($patterns, $placeholder, $contents);
+    if ($masked === NULL) {
+      throw new RuntimeException('Failed to mask secret token patterns in cast');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '339', column '17'). (undefined)

(MissingImport)